pkg/internal/templates.go

```diff
@@ -108,17 +108,8 @@ const commonTempl = `
 package {{.Version}}
 
 import (
-	"context"
-	"fmt"
-	{{.Name}}api{{.Version}} "{{.APIPath}}"
-	{{.Name}}{{.Version}} "{{.ClientPath}}/typed/{{.Name}}/{{.Version}}"
-
-	kcp "github.com/kcp-dev/apimachinery/pkg/client"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Since this is part of `appendGenericImports`, do I understand correctly that these are always imported? Why move this logic into Go from the template?

Reply:
Yes, just to have everything added in a same place. Could be either ways.